web/src/routes/(user)/explore/+page.svelte

```diff
@@ -36,6 +38,18 @@
       MAX_PLACE_ITEMS = screenSize < 768 ? Math.floor(innerWidth / 150) : Math.floor(innerWidth / 172);
     }
   }
+
+  onMount(() => {
+    return websocketEvents.on('on_person_thumbnail', (personId: string) => {
+      people.map((person) => {
+        if (person.id === personId) {
+          person.updatedAt = Date.now().toString();
```

Comment:
Shouldn't this be `new Date().toISOString()`?

Reply:
This works too, the main objective is to have a different value